packages/next-auth/src/providers/github.ts

```diff
@@ -98,6 +98,22 @@ export default function Github<P extends GithubProfile>(
         image: profile.avatar_url,
       }
     },
+    style: {
```

Comment:
I think we should rather accept a link and host the defaults in our repo next to the providers, maybe. Ideally we should make sure the logo sizes are the same as well.
Can we have some strict guidelines like transparent no shadow backgrounds required (saw this on the Google icon)? Then one logo would be enough too. optionally have a colored and monochrome version and add a toggle in `theme.signInButtons: "colored"|"monochrome" |"none"`

Reply:
Yeah so I was debating, for example with the Github one, you could reuse the same svg and then just adjust the `fill` color based on `light | dark` mode.

If we're just saving the links to svgs (in the repo for example) then I also don't think it'd be a big deal to allow options like `colored | monochrome | none`, etc. Dig it!

And regarding icon sizes and stricter requirements. I think these might be a good starting point, thoguhts?
- 32x32
- No whitespace around the svg (so the logo itself really is 32x32)
- ?

Reply:
The thing with the using the same svg file and adjusting it for light/dark automatically is, its going to be different for each provider, right? So we'd have to provide some more config options/fields where people can specify which color the `fill` should be in light/dark mode. And what if theres multiple shapes / paths that need different fills for each theme.. I think we might just be better off providing two (or more) svgs. Like `apple.svg` and `apple-dark.svg`.